### PR TITLE
Updating to allow for dynamically generated env.conf file to expose ...

### PR DESCRIPTION
**Pull request by ewolinetz**
all container environment variables to mod_perl

### CODE REVIEW

**Comment by ewolinetz** [comment]
@bparees PTAL, post 3.0 candidate

**Comment by bparees** [comment]
seems reasonable enough to me, did you do any research to see if there were any warnings about why not to do this?

**Comment by ewolinetz** [comment]
probably not enough, from what I've gathered so far by default mod_perl automatically allows a few environment variables and anything past those need to be specified with PerlPassEnv or PassEnv.  The difference being at what point they are exposed to Perl, with PerlPassEnv exposing it as soon as possible, the former is exposed after the fixup phase (http://perl.apache.org/docs/1.0/guide/config.html#PerlSetEnv_and_PerlPassEnv)

File: 5.16/.sti/bin/run [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 set -e
 
+exec env | awk -F'=' '{print "PerlPassEnv "$1}' >> /opt/openshift/config/env.conf
```
Comment (mnagy) [inline 5.16/.sti/bin/run]: s/exec //

**Comment by mnagy** [comment]
LGTM

**Comment by mfojtik** [comment]
LGTM although security me is screaming now :-)

File: 5.16/.sti/bin/run [hunk 2]
```diff
@@ -2,4 +2,6 @@
 
 set -e
 
+env | awk -F'=' '{print "PerlPassEnv "$1}' >> /opt/openshift/config/env.conf
```
Comment (bparees) [inline 5.16/.sti/bin/run]: please throw a warning comment before this explaining that it's passing all env through to mod_perl.

**Comment by bparees** [comment]
one nit and lgtm.

**Comment by ewolinetz** [comment]
Addressed comment

**Pull request merged**